src/LevelSets/compute_level_sets.jl

```diff
@@ -1,7 +1,3 @@
-using FastIce
-using FastIce.Grids
-using KernelAbstractions
-
 "Initialize level sets."
```

Comment:
If adding docstrings, check out to add them in the way we did it for others, i.e., including function signature and short description